FlowCrypt/Functionality/Pgp/KeyMethods.swift

```diff
@@ -35,17 +35,13 @@ final class KeyMethods: KeyMethodsType {
                 return nil
             }
 
-            guard let decrypted = try? self.decrypter.decryptKey(armoredPrv: privateKey, passphrase: passPhrase) else {
+            do {
+                _ = try self.decrypter.decryptKey(armoredPrv: privateKey, passphrase: passPhrase)
+                return key
+            } catch {
                 logger.logInfo("Filtered not decrypted key")
                 return nil
             }
```

Comment:
Looks good, but we should be careful to specifically catch the "pass phrase mismatch" kind of error, and still do something meaningful with other kind of errors, at a minimum log them. (but better, show to user as a modal)

Filing for another issue.